Source/Modules/php.cxx

```diff
@@ -1288,8 +1328,11 @@ class PHP : public Language {
        */
       if (Cmp(strrchr(GetChar(n, "sym:name"),'_'),"_set") == 0)
         static_setter = true;
-      else
-        static_getter = true;
+      else if (Cmp(strrchr(GetChar(n, "sym:name"),'_'),"_get") == 0) {
```

Comment:
I assume you're trying to test if the name ends in "_set" (or "_get") above?

The code doesn't actually do that though - it'll match for names containing "_set" or "_get" anywhere in them, which means that `num_settlers_get` be misidentified as a setter.

I'd suggesting adapting this existing code which does a very similar check correctly:

```
      const char *p = Char(iname);
      if (strlen(p) > 4) {
        p += strlen(p) - 4;
        String *varname = Getattr(n, "membervariableHandler:sym:name");
        if (strcmp(p, "_get") == 0) {
        } else if (strcmp(p, "_set") == 0) {
          Setattr(shadow_set_vars, varname, iname);
        }
      }
```